src/abstractitensornetwork.jl

```diff
@@ -870,6 +877,67 @@ function ITensors.commoninds(tn1::AbstractITensorNetwork, tn2::AbstractITensorNe
   return inds
 end
 
+"""Check if the edge of an itensornetwork has multiple indices"""
+is_multi_edge(tn::AbstractITensorNetwork, e) = length(linkinds(tn, e)) > 1
```

Comment:
Something neat you can do is define a single-argument version:
```julia
is_multi_edge(tn::AbstractITensorNetwork) = Base.Fix1(is_multi_edge, tn)
```
Then, `is_multi_edge(tn)(e)` is the same as `is_multi_edge(tn, e)`. This can simplify code where you want to pass `is_multi_edge` as a map or filter function, for example:
```julia
filter(is_multi_edge(tn), edges(tn))
```

Reply:
Oh nice! That's cool. I have added that now.